docs/requirements.md

```diff
@@ -2,88 +2,76 @@
 
-Check the
-[values-production.yaml](https://github.com/codacy/chart/blob/master/codacy/values-production.yaml)
-file to find a configuration reference that should work for you to run a
-"production-ready" version of Codacy.
+!["High Level Architecture"](<./images/High Level Architecture - Analysis II.svg> "High Level Architecture")
 
-### Analysis
+### Virtual CPUs and memory
 
-Each analysis runs a maximum number of 4 plugins in parallel (not configurable)
+Since all components are running on Kubernetes, you can increase the number of replicas in every deployment to give you more resilience and throughput, at a cost of increased resource usage. The following is a simplified overview of how to calculate resource allocation for the "Platform" and the "Analysis":
 
-Note: All the following configurations are nested inside the `worker-manager.config`
-configuration object, but for simplicity, we decided to omit the full path.
+| Component                          | vCPU                    | Memory                   |
+| ---------------------------------- | ----------------------- | ------------------------ |
+| Platform (1 replica per component) | 4                       | 8                        |
+| Analysis Worker                    | 5 (per Analysis Worker) | 10 (per Analysis Worker) |
 
-```text
-CPU: workerResources.requests.cpu + (pluginResources.requests.cpu * 4)
+The resources described on the following table are based on our experience and are also the defaults in the [values-production.yaml](https://raw.githubusercontent.com/codacy/chart/master/codacy/values-production.yaml) file, which you might need to adapt taking into account your use case.
 
-Memory: workerResources.requests.memory + (pluginResources.requests.memory * 4)
+| Installation type                        | ~ Total vCPUs | ~ Total Memory | Replicas per component | Max. commits analyzed concurrently | Platform vCPUs | Platform Memory | Analysis Workers vCPUs | Analysis Workers Memory |
+| ---------------------------------------- | ------------- | -------------- | ---------------------- | ---------------------------------- | -------------- | --------------- | ---------------------- | ----------------------- |
+| Kubernetes Small Installation            | 16            | 32 GB          | 1                      | 2                                  | 4              | 8 GB            | 10                     | 20 GB                   |
+| Kubernetes Medium Installation (default) | 32            | 64 GB          | 2                      | 4                                  | 8              | 16 GB           | 20                     | 40 GB                   |
+| Kubernetes Big Installation              | 60+           | 110+ GB        | 2+                     | 10+                                | 8+             | 16+ GB          | 50+                    | 100+ GB                 |
 
-Number of concurrent analysis: workers.dedicatedMax
-```
+**NOTE:**
```

Comment:
```suggestion
#### Microk8s Installation
**NOTE:**
```

Reply:
We are putting a note like it is across different sections. I think @pauloribeiro-codacy is thinking on a way to have all this sections more clear, but right now I think it is better to keep all the same